app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -1039,6 +1043,12 @@ class VaultAddEditViewModel @Inject constructor(
         }
     }
 
+    private fun handleLoginClearFido2Credential() {
+        updateLoginContent { loginType ->
+            loginType.copy(fido2CredentialCreationDateTime = null)
```

Comment:
Although this removes the creation timestamp for that `cipherView`, the passkey itself still exists in the `cipher`. To remove the actual passkey from the `cipher`, we must clear `cipher.login.fido2Credentials` list. `VaultAddItemStateExtensions.toLoginView` is where we convert the content view into a `Cipher` for updating. That is where you'll want to clear the list if `it.fido2CredentialCreationDateTime` is null.

Reply:
That makes sense, thank you!